.travis.yml

```diff
@@ -3,6 +3,9 @@ language: python
 # Use the new container-based Travis infrastructure.
 sudo: false
 
+# Make sure we are on Ubuntu 14.04
+dist: trusty
```

Comment:
The main reason was to make sure we stay on trusty and don't flip flop until trusty is default. Also this makes sure anyone running the tests on their personal repo will get the same results there and here.
I'll fix the commit-message.